packages/api/src/controllers/sensorController.js

```diff
@@ -221,11 +221,13 @@ const sensorController = {
           if (curr.status === 'fulfilled') {
             prev.push(curr.value);
           } else {
+            // These are sensors that were not saved to the database as locations
             errorSensors.push({
               row: data.findIndex((elem) => elem === registeredSensors[idx]) + 2,
-              column: 'External_ID',
```

Comment:
@kathyavini Did you mean to remove this?

It is now leaving the Column part blank in the errors text:

<img width="744" alt="Screenshot 2023-05-10 at 4 37 10 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/24cfc909-d5d3-40c4-86fd-ba2bc8c0291e">

I am guessing it was removed because it is not the core problem -- but it is probably good to keep the data point if it will be because of External ID once 3142 is fixed?.

Reply:
Yup! Super intentional (see comments in the Jira ticket).

The `external_id` is not at fault if the record fails to save to the database -- as far as I can gather this code block was just copied verbatim from further up in the controller where the errors were related to sensor registration; it's not like that here and I didn't change how the error is generated if there is an actual registration issue. We could adjust the error text template to delete the Column part when one isn't present, but that's going to touch on the translation files. Given that we're not going to see this very much in the future, I strongly feel it's not worth the effort. The blank column indicates that no column is at fault, which is correct 🙂

Reply:
To your other point people may continue being confused if we do see it in the future. I think a language translation or removal of the column field should not be off the table. 

Probably we have something like "not specified" or "unknown" already translated.

Reply:
I asked Kevin to be the final judge on this one, and he has cast his vote for blank column!

Reply:
Ok cool made a note on the jira ticket about our discussion. And will approve as is.